components/file_upload/file_upload.jsx

```diff
@@ -505,7 +508,9 @@ export default class FileUpload extends PureComponent {
 
                 const name = formatMessage(holders.pasted) + d.getFullYear() + '-' + (d.getMonth() + 1) + '-' + d.getDate() + ' ' + hour + '-' + minute + ext;
 
-                files.push(new File([file], name));
+                const newFile = new Blob([file], {type: file.type});
```

Comment:
Just to double check, the Blob constructor works in Edge? The MDN has it listed as "compatibility unknown" ([link](https://developer.mozilla.org/en-US/docs/Web/API/Blob#Browser_compatibility))

Reply:
Thank you for nudging on this. This change does resolve the issue for Edge. 

In double checking now, however, I realized that in IE, the paste of an image will now result in its base64 encoded string. 

Not sure if that should be tackled separately or hold up the PR. I’ll need to investigate further to determine an IE workaround.

Reply:
That sounds like it would block this PR. Aside from fixing it, we could consider not supporting pasting files on IE11. Thoughts on doing that, @esethna?